course/test/src/Ant_TEST.cc

```diff
@@ -0,0 +1,129 @@
+#include <cmath>
+#include <sstream>
+#include <string>
+#include <memory>
+
+#include "gtest/gtest.h"
+
+#include "ant.h"
+#include "cell_mock.h"
+
+namespace ekumen {
+namespace simulation {
+namespace test {
+namespace {
+
+GTEST_TEST(AntTest, OccupiedCellOnInitialization) {
+  std::shared_ptr<Cell> cell(new Cell());
+  Ant ant(cell);
+  EXPECT_FALSE(cell->IsFree());
+}
+
+GTEST_TEST(AntTest, CallbackAssignedCorrectly) {
+  std::shared_ptr<Cell> cell(new Cell());
+  Ant ant = Ant(cell);
+  bool lambda_called = false;
+  auto callback = [&](InsectCallbackMetrics& metrics) {
```

Comment:
you don't need to capture the full context, you can just capture the variable you need `lambda_called`

Reply:
Good point. Done.